### PR TITLE
Added Note for 404 error regarding /robots.txt files

### LINKED ISSUE #5237: Contact support when robots.txt 404s
# General issue

## Description:

I'd like to make a suggestion for https://devdocs.magento.com/guides/v2.2/cloud/trouble/robots-sitemap.html.

## Possible solutions:

Add a troubleshooting section saying that if your `robots.txt` file goes to 404, you should open a Support Ticket requesting to remove the redirect from `/robots.txt to /media/robots.txt`
https://magento.zendesk.com/agent/tickets/238834

Alternatively, to add that bit to the line that says You do not have to generate a robots.txt because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the url: `<domain.your.project>/robots.txt`
basically, if `<domain.your.project>/robots.txt` gives a 404 error, to open a Support ticket

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) adds a note regarding /robots.txt errors telling users to open a Support ticket.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/cloud/trouble/robots-sitemap.html

fixes #5237

### CODE REVIEW

File: guides/v2.2/cloud/trouble/robots-sitemap.md [hunk 1]
```diff
@@ -7,43 +7,47 @@ functional_areas:
 ---
 An attempt to generate and write the `sitemap.xml` file to the root directory results in the following error:
 
-```
+```bash
```
Comment (jeff-matthews) [inline guides/v2.2/cloud/trouble/robots-sitemap.md]:
```suggestion
```terminal
```

Comment (jeff-matthews) [reply guides/v2.2/cloud/trouble/robots-sitemap.md]: Use "bash" for actual commands and "terminal" for console output.

Comment (hickskenh) [reply guides/v2.2/cloud/trouble/robots-sitemap.md]: Thank you.  first time i've seen that linting error.  corrected and committed 2da31df

File: guides/v2.2/cloud/trouble/robots-sitemap.md [hunk 2]
```diff
@@ -7,43 +7,47 @@ functional_areas:
...
-You do not have to generate a `robots.txt` because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the url: `<domain.your.project>/robots.txt`
+You do not have to generate a `robots.txt` because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the `<domain.your.project>/robots.txt` file.
 
 This requires ECE-Tools version 2002.0.12 and later with an updated `.magento.app.yaml` file. See an example of these rules in the [magento-cloud repository](https://github.com/magento/magento-cloud/blob/master/.magento.app.yaml#L43-L49).
 
-#### To generate a `sitemap.xml` file in version 2.2 and later:
+#### To generate a `sitemap.xml` file in version 2.2 and later
+
+1. Access the Magento Admin panel.
+1. On the _Marketing_ menu, click **Site Map** in the _SEO & Search_ section.
```
Comment (shrielenee) [inline guides/v2.2/cloud/trouble/robots-sitemap.md]: I assume italicizing UI areas you don't click on is a cloud style thing?

File: guides/v2.2/cloud/trouble/robots-sitemap.md [hunk 3]
```diff
@@ -7,43 +7,47 @@ functional_areas:
...
-1.  On the _Marketing_ menu, click **Site Map** in the _SEO & Search_ section.
-1.  In the _Site Map_ view, click **Add Sitemap**.
-1.  In the _New Site Map_ view, enter the following values:
+   -  **Filename**:`sitemap.xml`
+   -  **Path**:`/media/`
 
-    -  **Filename**:`sitemap.xml`
-    -  **Path**:`/media/`
+1. Click **Save & Generate**. The new site map becomes available in the _Site Map_ grid.
+1. Click the path in the `Link for Google` column.
```
Comment (shrielenee) [inline guides/v2.2/cloud/trouble/robots-sitemap.md]: Why is this in backticks, but other UI areas are in italics?

File: guides/v2.2/cloud/trouble/robots-sitemap.md (hunk 2 above)
Comment (hickskenh) [reply guides/v2.2/cloud/trouble/robots-sitemap.md]: when doing a search for "menu,", i found other references to menus (Marketing x3, Extension x2, Content x2, Account x2). Marketing and Content have the ital formatting, Account and Extension do not.

File: guides/v2.2/cloud/trouble/robots-sitemap.md (hunk 3 above)
Comment (hickskenh) [reply guides/v2.2/cloud/trouble/robots-sitemap.md]: in a search for " column", i found instances where the name of a column is in `s, in bold, and with no formatting.

File: guides/v2.2/cloud/trouble/robots-sitemap.md (hunk 2 above)
Comment (meker12) [reply guides/v2.2/cloud/trouble/robots-sitemap.md]: That's the style in Cloud, but I know that it is not consistently applied.

File: guides/v2.2/cloud/trouble/robots-sitemap.md (hunk 3 above)
Comment (meker12) [reply guides/v2.2/cloud/trouble/robots-sitemap.md]: Aiming for consistency within a topic, the other column names in the procedure are in italics.

File: guides/v2.2/cloud/trouble/robots-sitemap.md (hunk 2 above)
Comment (shrielenee) [reply guides/v2.2/cloud/trouble/robots-sitemap.md]: There has been much discussion about what to do here. It's ongoing... I just wish we were consisten across repos/doc sets!

File: guides/v2.2/cloud/trouble/robots-sitemap.md (hunk 3 above)
Comment (hickskenh) [reply guides/v2.2/cloud/trouble/robots-sitemap.md]: Changing  Link for Google column name to ital.

File: guides/v2.2/cloud/trouble/robots-sitemap.md (hunk 2 above)
Comment (hickskenh) [reply guides/v2.2/cloud/trouble/robots-sitemap.md]: leaving menu titles in ital for this topic

**Comment by hickskenh** [comment]
running tests

**Pull request merged**